Core/src/org/sleuthkit/autopsy/modules/embeddedfileextractor/ImageExtractor.java

```diff
@@ -211,6 +213,13 @@ void extractImage(AbstractFile abstractFile) {
         HWPFDocument doc = null;
         try {
             doc = new HWPFDocument(new ReadContentInputStream(af));
+        } catch (OldFileFormatException ex) {
```

Comment:
No, I don't think we should log those, either. This is a best effort endeavor, and the WARNINGS you mentioned are in the same category as the others, i.e., not really helping anybody, just inflating the logs.

However, the TskCoreException catches where it is a database issue should log SEVEREs, not WARNINGs. For example, the TskCoreException catch in isImageExtractionSupported and the AbstractFile.hasChildren call in extractImage.